api/api/spec/spec.yaml

```diff
@@ -19066,6 +19076,67 @@ paths:
         '429':
           $ref: '#/components/responses/TooManyRequestsResponse'
 
+  /events:
```

Comment:
We need to ensure that the 100 bulk size limit and the 30 max POST events EPS are explained somewhere in the documentation!